TargetComposition/SymBuilder/SymbolTypes.cpp

```diff
@@ -42,8 +42,9 @@ HRESULT UdtPositionalSymbol::BaseInitialize(_In_ SymbolSet *pSymbolSet,
     // the actual offset the same as the offset.  Any triggering of type layout will change this if such
     // is marked as automatic.
     //
+    HRESULT hr = BaseDataSymbol::BaseInitialize(pSymbolSet, symKind, owningTypeId, symOffset, symTypeId, pwszName, nullptr);
     m_symOffsetActual = m_symOffset;

```

Comment:
>  m_symOffsetActual = m_symOffset; [](http://example.com/codeflow?start=2&length=34)

*MUST FIX*: This breaks automatic layout of the type.  Manual layout from import or explicit API works fine.

We should add some unit tests to this at some point soon...

I'll fix this shortly in a subsequent push.